From e4a7054bef4f664ac0f40946d3a067df78430632 Mon Sep 17 00:00:00 2001
Message-Id: <patch-01.12-e4a7054bef4-20220224T092601Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Sat, 19 Feb 2022 22:23:37 +0100
Subject: [PATCH 01/12] imap-send.c: use designated initializers for "struct
 imap_server_conf"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Cut down a lot on the verbosity of the "server" assignment in
imap-send.c using designated initializers, only the "ssl_verify"
member was being set to a non-NULL non-0 value.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 imap-send.c | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/imap-send.c b/imap-send.c
index e6090a0346a..5ac6fa9c664 100644
--- a/imap-send.c
+++ b/imap-send.c
@@ -98,17 +98,7 @@ struct imap_server_conf {
 };
 
 static struct imap_server_conf server = {
-	NULL,	/* name */
-	NULL,	/* tunnel */
-	NULL,	/* host */
-	0,	/* port */
-	NULL,	/* folder */
-	NULL,	/* user */
-	NULL,	/* pass */
-	0,   	/* use_ssl */
-	1,   	/* ssl_verify */
-	0,   	/* use_html */
-	NULL,	/* auth_method */
+	.ssl_verify = 1,
 };
 
 struct imap_socket {
-- 
2.35.1.1157.g524e2d5a0db

